pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1800,6 +1805,10 @@ public void deleteRealtimeTable(String tableName) {
     HelixHelper.removeResourceFromBrokerIdealState(_helixZkManager, realtimeTableName);
     LOGGER.info("Deleting table {}: Removed from broker resource", realtimeTableName);
 
+    // Drop the table on servers
+    deleteTableOnServer(realtimeTableName);
+    LOGGER.info("Deleting table {}: Sent table deletion message to servers", realtimeTableName);
```

Comment:
We may not need to print this message as there is a similar message printed inside `deleteTableOnServer` method already. Plus, even inside the `deleteTableOnServer` method there is still some scenarios that the message doesn't need to be sent. So I'd suggest skipping this log here.

Reply:
Ah, make sense. Removed this line.